src/Apps/Auction/Components/AuctionDetails/AuctionInfoSidebar.tsx

```diff
@@ -39,6 +39,17 @@ const AuctionInfoSidebar: React.FC<AuctionInfoSidebarProps> = ({ sale }) => {
           <Text variant="sm">Supplemental Conditions of Sale</Text>
         </RouterLink>
       </Box>
+
+      <Box>
```

Comment:
Could also make this the parent `<Text variant="sm>`, then just use a `<br />` instead of 2 child elements. Does not really matter at all in this case. Sometimes we just have a lot of extra divs in grids.

Reply:
sure i can do that. izipizi!